recipes/source.rb

```diff
@@ -19,7 +19,15 @@
 
 include_recipe 'build-essential::default'
 
-user node['prometheus']['source']['user'] do
+if node['prometheus']['source']['use_existing_user'] == false
+  prometheus_user = node['prometheus']['source']['user']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.